tests/fixtures/migrations/await_precedence.php.in

```diff
@@ -0,0 +1,27 @@
+<?hh // strict
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+ async function foo(Awaitable<bool> $a, Awaitable<bool> $b): bool {
+  return await true ? $a : $b;
+ }
+
+ async function fooComments(Awaitable<bool> $a, Awaitable<bool> $b): bool {
+  return await /* for testing. */ true ? $a : $b;
+ }
+
+ async function bar(function (bool): Awaitable<void> $f, bool $x): void {
+     return await $x |> $f($$);
+ }
+
+  async function barComments(function (bool): Awaitable<void> $f, bool $x): void {
+     return await $x 
+        |> $f($$) // Comments!
+        ;
+ }
```

Comment:
Add newline at EOF; I guess you're intentionally not running hackfmt here as you're testing whitespace handling?

Reply:
No, I just rather assumed that VSCode would do that when I saved the file or closed a brace... running it by hand now.

Reply:
If you want that, command-shift-p, and 'Preferences: Open Settings (JSON)', and add:

```
    "editor.formatOnType": true,
    "files.insertFinalNewline": true,
    "files.trimTrailingWhitespace": true,
    "files.trimFinalNewlines": true,
```

Reply:
Ah, the await as expression meant that hackfmt couldn't parse it, so it wouldn't reformat it.